### PR TITLE
Improving community GitHub Actions workflows (Lombiq Technologies: OCORE-182)

### PR DESCRIPTION
**Pull request by Piedone**
- Fixing that the Close Stale PRs and Issues workflow frequently didn't finish, like [here](https://github.com/OrchardCMS/OrchardCore/actions/runs/9458375555) and needed either manual reruns, or waiting days until it has run sufficient times again.
- Linking to the [the docs](https://docs.orchardcore.net/en/latest/guides/contributing/managing-issues/#what-happens-after-you-open-an-issue) from auto comments for first-time issues [like this](https://github.com/OrchardCMS/OrchardCore/issues/16289#issuecomment-2160111377) to not leave people wondering [like this](https://github.com/OrchardCMS/OrchardCore/issues/16289#issuecomment-2161138920).

### CODE REVIEW

**Comment by Piedone** [comment]
@MikeAlhayek can you please review?

File: .github/workflows/close_stale_prs_issues.yml [hunk 1]
```diff
@@ -4,8 +4,10 @@ name: Close Stale PRs and Issues
 
 on:
   schedule:
-    # Every day at 1:39 AM UTC. A random time to avoid GitHub Actions being overloaded by scheduled workflows.
-    - cron: '39 1 * * *'
+    # Every four hours at :39 UTC. A random time to avoid GitHub Actions being overloaded by scheduled workflows. Need
+    # to run this relatively frequently to avoid "No more operations left! Exiting..." warnings. Increasing the operations
+    # (see https://github.com/actions/stale#operations-per-run) is risky.
+    - cron: '39 */4 * * *'
```
Comment (MikeAlhayek) [inline .github/workflows/close_stale_prs_issues.yml:10]: Why do you want it to run every 4 hours?

Comment (Piedone) [reply .github/workflows/close_stale_prs_issues.yml:10]: Read the comment above it and the PR description :).

Comment (MikeAlhayek) [reply .github/workflows/close_stale_prs_issues.yml:10]: I mean why 4 hours? Is there a significant value to 4 or can it run every 1, 3 or 6?

Comment (Piedone) [reply .github/workflows/close_stale_prs_issues.yml:10]: This seems like safely frequent enough for OC (i.e. 6 times a day), based on how many times I've seen workflow runs needed to be re-run before getting rid of "No more operations left! Exiting...".

There would be no harm in running it more frequently to an extent of like once per hour, apart from wasting a workflow runner's compute time for most runs (which for us is free, though obviously, we mustn't abuse this). Anything more frequent would most possibly cause it to occasionally fail due to too frequent API access, and would definitely be a waste.

Comment (MikeAlhayek) [reply .github/workflows/close_stale_prs_issues.yml:10]: Got it. Just wanted to make sure there is no real key of using 4.

**Review by MikeAlhayek — APPROVED** [review]

**Pull request merged**